src/main/java/org/betonquest/betonquest/quest/condition/realtime/RealTimeConditionFactory.java

```diff
@@ -0,0 +1,25 @@
+package org.betonquest.betonquest.quest.condition.realtime;
+
+import org.betonquest.betonquest.Instruction;
+import org.betonquest.betonquest.api.quest.condition.PlayerlessCondition;
+import org.betonquest.betonquest.api.quest.condition.PlayerlessConditionFactory;
+import org.betonquest.betonquest.exceptions.InstructionParseException;
+import org.betonquest.betonquest.quest.condition.time.TimeFrame;
```

Comment:
Should we may change packaging here? like 
```
- condition.time
  - .ingame
  - .real
```

I am unsure about this, as we currently have mainly for every e.g. event one package

Reply:
Yes, we should pack that together.